src/Models/Participant.php

```diff
@@ -70,6 +70,7 @@ public function getUser(): UserInterface
 
     public function email()
     {
+        //Todo: Determine relationship
```

Comment:
@drewroberts I don't think this should be a `hasOne` relationship. Correct me if I'm wrong, but if we're attaching an `email_address_id` to Participant, wouldn't the relationship be a `BelongsTo` since an email address does not have a `particpant_id` column?

Reply:
Yes, that is correct.

Reply:
https://github.com/tipoff/bookings/pull/88/commits/8034157d20f39c854747570bb00bb70104ee2e5c